.gitignore

```diff
@@ -6,3 +6,4 @@
 venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
```

Comment:
There is an extra space at the end of the line for `db.sqlite3`. This might prevent it from being ignored properly. Consider removing the space to ensure it is correctly ignored by Git.